README.md

```diff
@@ -18,288 +18,20 @@ Azure Key Vault provider for [Secrets Store CSI driver](https://github.com/kuber
 - Supports sync with Kubernetes Secrets (Secrets Store CSI Driver v0.0.10+)
 - Supports multiple secrets stores providers in the same cluster.
 
-#### Table of Contents
+## Getting started
 
-- [Demo](#demo)
-- [Usage](#usage)
-  - [Install with Helm](#install-the-secrets-store-csi-driver-and-the-azure-keyvault-provider)
-  - [Using the Azure Key Vault Provider](#using-the-azure-key-vault-provider)
-- [Azure Key Vault Provider Features](#azure-key-vault-provider-features)
-- [Troubleshooting](#troubleshooting)
-- [Contributing](#contributing)
-- [Testing](#testing)
-- [Other Azure Clouds](#other-azure-clouds)
-- [Support](#support)
+Setup the correct [role assignments and access policies](https://azure.github.io/secrets-store-csi-driver-provider-azure/configurations/identity-access-modes/) and install Azure Keyvault Provider for Secrets Store CSI Driver through [Helm](https://azure.github.io/secrets-store-csi-driver-provider-azure/getting-started/installation/#deployment-using-helm) or [YAML deployment files](https://azure.github.io/secrets-store-csi-driver-provider-azure/getting-started/installation/#using-deployment-yamls). Get familiar with [how to use the Azure Keyvault Provider](https://azure.github.io/secrets-store-csi-driver-provider-azure/getting-started/usage/) and supported [configurations](https://azure.github.io/secrets-store-csi-driver-provider-azure/configurations/).
 
-## Demo
-
-![Azure Key Vault Provider for Secrets Store CSI Driver Demo](images/demo.gif "Secrets Store CSI Driver Azure Key Vault Provider Demo")
```

Comment:
I like the demo in the main readme unless it is out-of-date. @phillipgibson @ritazh wdyt?

Reply:
I've moved the demo to the site, but I like the idea of just keeping it in the readme too.

Reply:
+1 keep it

Reply:
Added it back to the readme.